### ISSUE TITLE
[HDF5] Prevent clicking on last scan if on-going scan

### ISSUE THREAD

**Issue by vasole** [vasole/pymca#834]
ID26 would like to avoid crashes because of accessing the last scan.

Still not clear what to do:

- missing end_time?
- checking last time the file was modified?

**Comment by woutdenolf** [comment]
Segmentation faults are most likely not coming from accessing the last scan. You get error popups from accessing the last scan.

I've seen segfaults only when iterating over the top-level items of an HDF5 file that is being written to. The solution is to get the top-level names in a subprocess. Ones you have those you will be able to access the file without segfaults (but you still could get exceptions, especially for the last scan). I'll make a MR to do this in the HDF5 Qt filemodel.

**Comment by vasole** [comment]
You are right. We tested it on ID26 and segfaults were only appearing when first opening the file.

**Comment by vasole** [comment]
PR #848 made this not necessary
